.github/workflows/bot.yml

```diff
@@ -9,6 +9,8 @@ on:
     branches:
       - master
       - 'release-*'
+env:
+  MVN_ARGS: -ntp -B -V -Pwarn-log -Dorg.slf4j.simpleLogger.log.org.apache.maven.plugins.shade=warn -Dorg.slf4j.simpleLogger.log.org.apache.maven.plugins.dependency=warn
```

Comment:
Can we add these overrides to the config instead of needing to specify them here?

Reply:
For the shade and dependency plugin, I didn't find a way to specify the log level in the configuration of the plugin unfortunately. We could put these in a basic logging configuration file that maven uses as well though.

For the `-ntp -B -V`, that will need to be here

Reply:
Interesting. You're saying if you specify the logs it in props for these loggers it's not not picked up?

Reply:
We can specify a logging configuration for maven itself if we want and configure the simple logger there

Reply:
I think that would be preferred

Reply:
I've tried debugging this for a couple of hours but it looks like maven will not pick up our properties file so I am going to revert back to manually specifying the strings in the build file